pkgs/top-level/all-packages.nix

```
@@ -14127,10 +14127,7 @@ with pkgs;
 
   lttv = callPackage ../development/tools/misc/lttv { };
 
-  luaformatter = callPackage ../development/tools/luaformatter
-    (lib.optionalAttrs stdenv.isDarwin {
-      stdenv = overrideCC stdenv llvmPackages_latest.clang;
-    });
+  luaformatter = callPackage ../development/tools/luaformatter { };
```

I think you can fix the build with:
```suggestion
  luaformatter = callPackage ../development/tools/luaformatter {
    stdenv = if stdenv.cc.isClang then llvmPackages_9.stdenv else stdenv;
  };
```

---

it works! thanks

weird that llvmPackages_latest doesn't work